src/redux-middleware/scrollCursorIntoView.ts

```diff
@@ -19,6 +24,30 @@ editingValueStore.subscribe(
   }, 400),
 )
 
+// store the previous isPulling value, to only invoke scrollCursorIntoView when it changes to `false`
+let prevIsPulling: boolean | null = null
+/**
+ * Scroll the cursor into view if the sync was the result of navigation to a thought.
+ */
+syncStatusStore.subscribe(state => {
+  if (!state.isPulling && prevIsPulling && navigated) {
```

Comment:
Any reason not to call `scrollCursorIntoView` whenever `isPulling` changes to false, regardless of `navigated`? It would reduce the complexity.

Reply:
That would basically tie pulling to scrolling, not sure if there are cases in which we don't want to force a scroll, e.g. when loading favorites?